client/payment-methods-map.ts

```diff
@@ -16,7 +16,20 @@ import P24Icon from './gateway-icons/p24';
 import IdealIcon from './gateway-icons/ideal';
 import BankDebitIcon from './gateway-icons/bank-debit';
 
-export default {
+export interface PaymentMethodMapEntry {
+	id: string;
+	label: string;
+	description: string;
+	// eslint-disable-next-line @typescript-eslint/naming-convention
+	Icon: () => JSX.Element;
```

Comment:
Not sure, if this is a leftover.

Reply:
Good catch. I removed this, but it was a little tricky. It's being used in multiple non-TS files, so I made a global search and changed it everywhere I can find it.